lib/managers/webhooks.js

```diff
@@ -0,0 +1,138 @@
+/**
+* @Webhookoverview Manager for the Box Webhooks Resource
+* @author ccheng
+*/
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+var urlPath = require('../util/url-path');
+
+// ------------------------------------------------------------------------------
+// Private
+// ------------------------------------------------------------------------------
+
+// Base path for all webhooks endpoints
+var BASE_PATH = '/webhooks';
+
+// ------------------------------------------------------------------------------
+// Public
+// ------------------------------------------------------------------------------
+
+/**
+* Simple manager for interacting with all 'Webhooks' endpoints and actions.
+*
+* @param {BoxClient} client The Box API Client that is responsible for making calls to the API
+* @constructor
+*/
+function Webhooks(client) {
+	// Attach the client, for making API calls
+	this.client = client;
+}
+
+
+/**
+* Returns a webhook object if creation succeeds.
+*
+* API Endpoint: '/webhooks'
+* Method: POST
+*
+* @param {string} id - File or Folder ID to create webhook on
+* @param {string} type - Value of 'file' or 'folder'
+* @param {string} notificationURL - The url of your application where Box will notify you of events triggers
+* @param {string[]} triggerTypes - Array of event types that trigger notification for the target
```

Comment:
@mattwiller do you mean something like this? 

`var eventTriggers = {
	fileUploaded: 'FILE.UPLOADED',
	filePreviewed: 'FILE.PREVIEWED',
	fileDownloaded: 'FILE.DOWNLOADED',
	fileTrahed: 'FILE.TRASHED'
	...
}`

?? I also noticed that files and folders have some differences in allowed trigger types. Wandering if we should put an eventTrigger enum in the files manager/ folders manager for createWebhook() as well as in the webhooks manager for the rest of the calls?